changelog.d/1022.breaking.md

```diff
@@ -0,0 +1,2 @@
+The `parse_groks` VRL function has the multiline mode enabled by default.
```

Comment:
I'm not quite sure whether it makes sense to introduce an optional parameter for users to control this or simply keep existing behaviour for this function? 🤔  Datadog's grok implementation enabled multiline mode by default, which seems like a reasonable behaviour. Also users they can always use `(?m)` and `(?-m)` modifiers to control this behaviour.
cc @bruceg @jszwedko

Reply:
I don't have as much context on `vrl` but my 2cents - agreed that making this the default behavior feels reasonable, and I think it would also make sense to eventually update `parse_grok` to have parity. If I'm looking at the [vrl docs](https://vector.dev/docs/reference/vrl/functions/#parse_grok) I would assume they share this base behavior.